terraform/modules/gcs_source/main.tf

```diff
@@ -6,7 +6,7 @@ locals {
 
 data "archive_file" "source" {
   type        = "zip"
-  source_dir  = "${path.root}/.."
+  source_dir  = "${path.root}/${var.source_folder_relative_path}"
```

Comment:
In this file it only signals this change, but other files have problems in that regard.